### PR TITLE
The use of mixed quotes for src= is not intuitive

### PR DESCRIPTION
**Pull request by rickatech**
Providing example here will help avoid such confusion!

### CODE REVIEW

File: src/v2/api/index.md [hunk 1]
```diff
@@ -1518,6 +1518,7 @@ All lifecycle hooks automatically have their `this` context bound to the instanc
 
   <!-- shorthand -->
   <img :src="imageSrc">
+  <img v-bind:src="'/storage/userimages/' + item.imagefile">
```
Comment (kazupon) [inline src/v2/api/index.md]: Here is shorthand.
If you add the mixed quotes, you should be added the above into example.

Comment (rickatech) [inline src/v2/api/index.md]: Or perhaps go with more concise pseudo prose ....

**Comment by chrisvfritz** [comment]
Tweaked it a bit. Thanks!

**Comment by rickatech** [comment]
:-)

**Pull request merged**